### PR TITLE
[15.0][ADD] fieldservice_equipment_note, fieldservice_equipment_note_sale_reference

### PR DESCRIPTION
**Pull request by LucPinheiro**
@ChristianSantamaria

### CODE REVIEW

File: fieldservice_equipment_note/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "name": "Fieldservice Equipment Note",
+    "summary": """
+        Add new fields in the fsm order models.
+    """,
+    "author": "Solvos",
+    "license": "AGPL-3",
+    "version": "15.0.1.0.0",
+    "category": "Field Service",
+    "website": "https://github.com/solvosci/slv-field-service",
+    "depends": ["fieldservice", "fieldservice_sale_reference"],
```
Comment (ChristianSantamaria) [inline fieldservice_equipment_note/__manifest__.py]:
```suggestion
    "depends": ["fieldservice_sale_reference"],
```

File: fieldservice_equipment_note/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+{
+    "name": "Fieldservice Equipment Note",
+    "summary": """
+        Add new fields in the fsm order models.
```
Comment (ChristianSantamaria) [inline fieldservice_equipment_note/__manifest__.py]:
```suggestion
        Adds new model to have equipment observations in each fsm.order.
```
Also change readme/description
Generate Readme again with new changes

**Review by ChristianSantamaria — CHANGES_REQUESTED** [review]
Some improvements:

File: fieldservice_equipment_note/reports/i18n/es.po [hunk 3]
```diff
@@ -0,0 +1,118 @@
+# Translation of Odoo Server.
```
Comment (ChristianSantamaria) [inline fieldservice_equipment_note/reports/i18n/es.po]: This folder was moved place put it where it belongs

File: fieldservice_equipment_note/README.rst [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-solvosci%2Fslv--field--service--actualizado-lightgray.png?logo=github
+    :target: https://github.com/solvosci/slv-field-service-actualizado/tree/15.0/fieldservice_equipment_note
+    :alt: solvosci/slv-field-service-actualizado
```
Comment (ChristianSantamaria) [inline fieldservice_equipment_note/README.rst]: The same as in the other PR
```suggestion
    :alt: solvosci/slv-field-service
```

**Review by ChristianSantamaria — CHANGES_REQUESTED** [review]
The last fixes:

**Comment by dalonsod** [comment]
@LucPinheiro tested and made several changes (see https://github.com/solvosci/slv-field-service/pull/19/commits/ff40832f8e978c1541b17e81d795fa3ea3dd0fa7 for check differences), please analyze and test them:

- Removed manifest dependency with `fieldservice_sale_reference` and made addon only dependent on `fieldservice`. Also removed `sale_reference` fields usages (*).
- Added description for `fsm.equipment.note` model.
- In general I've moved "Observation" typo to more correct "note", even in field definitions.
- For FSM Order, renamed `fsm_equipment_ids` by a more accurate `fsm_equipment_note_ids` name.
- Some translations adjustments. Removed `i18n` folder inside reports folder.
- Remove unnecessary `create="1"` in equipment notes tree inline view. Removed duplicated `editable="bottom"`.

(*) This change is due to separate functionality in tho different addons. Once `fieldservice_equipment_note` is ready, we should create a `fieldservice_equipment_note_sale_reference` addon that adds the `sale_reference` usages previously removed. We should add this new addon in another commit in this PR, and task will be finished

**Review by dalonsod — APPROVED** [review]
Functionality finally splitted in two addons, ready to be merged. Let's go...

**Pull request merged**